apps/web/src/services/cost-explorer/composables/data-source-converter.ts

```diff
@@ -0,0 +1,48 @@
+import type { ValueHandler } from '@spaceone/design-system/types/inputs/search/query-search/type';
+import { cloneDeep, keyBy } from 'lodash';
+
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+import type { ApiFilter } from '@cloudforet/core-lib/space-connector/type';
+
+import type { WorkspaceModel } from '@/schema/identity/workspace/model';
+
+export function convertDistinctValueHandler(
```

Comment:
suggestion: How about to change to `makeDataSourceDistinctValueHandler` or else..?